src/chapters/control-flow/exercises.rst

```diff
@@ -0,0 +1,47 @@
+Exercises: Control Flow
+=========================
+
+Work on these exercises in the IntelliJ ``java-web-dev-exercises`` project,
+creating a new class for each item. You may call these classes whatever you
+like, but remember to use the proper
+:ref:`Java naming conventions <naming-conventions>`.
+
+``Array`` Practice
+-------------------
+
+#. Create and initialize an array with the following values in a single line:
```

Comment:
My preference for styling on these type names is capitalized w/o syntax highlighting to help distinguish between the types that have a primitive and an object of the same name.

Reply:
Got it.  Fixed.